app/schemas.py

```diff
@@ -220,8 +220,6 @@ class ServiceSchema(BaseSchema, UUIDsAsStringsMixin):
     sms_message_limit = field_for(models.Service, "sms_message_limit", required=True)
     letter_message_limit = field_for(models.Service, "letter_message_limit", required=True)
     go_live_at = field_for(models.Service, "go_live_at", format=DATETIME_FORMAT_NO_TIMEZONE)
-    allowed_broadcast_provider = fields.Method(dump_only=True, serialize="_get_allowed_broadcast_provider")
-    broadcast_channel = fields.Method(dump_only=True, serialize="_get_broadcast_channel")
```

Comment:
And there's not some component of ours still supplying this value in its api calls and we're not strict enough to reject it if we get unknown keys...?